publisher/product/glossary/build.sh

```diff
@@ -115,390 +31,65 @@ function publishProductGlossaryRemoveWarnings() {
-  local glossaryBaseName="$2"
-
-  log "Creating Excel file from $(logFileName "${glossaryBaseName}.csv")"
-
-  #
-  # Set the memory for ARQ
-  #
-  export JVM_ARGS="${JVM_ARGS:--Xmx4G}"
-
-  cat > "${TMPDIR}/makeCcsv.sparql" << __HERE__
-PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
-PREFIX owlnames: <http://spec.edmcouncil.org/owlnames#>
-PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
-PREFIX owl: <http://www.w3.org/2002/07/owl#>
-PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
-PREFIX av: <https://spec.edmcouncil.org/fibo/ontology/FND/Utilities/AnnotationVocabulary/>
-
-SELECT ?Term ?Type (GROUP_CONCAT (?syn; separator=",") AS ?Synonyms) ?Definition ?GeneratedDefinition  ?example ?explanatoryNote ?ReleaseStatus
-WHERE {
-  ?c  av:hasMaturityLevel ?level ;a ?metaclass  .
-  BIND (IF ((?level=av:Release), "Production", "Development") AS ?ReleaseStatus)
-  FILTER (REGEX (xsd:string (?c), "edmcouncil"))
-  ?c  owlnames:definition ?Definition ;
-  owlnames:label ?Term .
-  FILTER (?Term != "")
-
-  ?metaclass owlnames:label ?Type . 
+	logRule "Collecting DEV and PROD ontologies"
+	
+	${PYTHON3} ${SCRIPT_DIR}/lib/fibos_collector.py --input_folder /input/fibo --output_dev ${TMPDIR}/dev.rdf --output_prod ${TMPDIR}/prod.rdf 
+  
+	if [ ${PIPESTATUS[0]} -ne 0 ] ; then
+		error "Could not collect FIBO ontologies"
+		return 1
+	fi
+  
+	logRule "Creating data dictionaries for DEV and PROD"
+  
+	#
+	# Set the memory for ARQ
+	#
+	JVM_ARGS="--add-opens java.base/java.lang=ALL-UNNAMED"
+	JVM_ARGS="${JVM_ARGS} -Dxxx=arq"
+	JVM_ARGS="${JVM_ARGS} -Xms2g"
+	JVM_ARGS="${JVM_ARGS} -Xmx4g"
+	JVM_ARGS="${JVM_ARGS} -Dfile.encoding=UTF-8"
+	JVM_ARGS="${JVM_ARGS} -Djava.io.tmpdir=\"${TMPDIR}\""
+	export JVM_ARGS
+	logVar JVM_ARGS
+	
+	${JENA_ARQ} --data ${TMPDIR}/dev.rdf --query ${SCRIPT_DIR}/product/glossary/data_dictionary.sparql --results=CSV > ${TMPDIR}/dev_proto_data_dictionary.csv
+	${JENA_ARQ} --data ${TMPDIR}/prod.rdf --query ${SCRIPT_DIR}/product/glossary/data_dictionary.sparql --results=CSV > ${TMPDIR}/prod_proto_data_dictionary.csv
+        
+	logRule "Extending data dictionaries with generated definitions"
+		
+	${PYTHON3} ${SCRIPT_DIR}/lib/dictionary_maker_no_sparql.py --input ${TMPDIR}/dev_proto_data_dictionary.csv --output ${glossary_product_tag_root}/glossary-dev.csv
+  	${PYTHON3} ${SCRIPT_DIR}/lib/dictionary_maker_no_sparql.py --input ${TMPDIR}/prod_proto_data_dictionary.csv --output ${glossary_product_tag_root}/glossary-prod.csv
+  
+	logRule "Writing from csv files to xlsx files"
 
-  OPTIONAL {?c owlnames:synonym ?syn}
-  OPTIONAL {?c owlnames:example ?example}
-  OPTIONAL {?c owlnames:explanatoryNote ?explanatoryNote}
+	touch "${glossary_product_tag_root}/glossary.log"	
 
-  OPTIONAL {?c  owlnames:mdDefinition ?GeneratedDefinition}
-}
-GROUP BY ?c ?Term ?Type ?Definition ?GeneratedDefinition ?example ?explanatoryNote ?ReleaseStatus
-ORDER BY ?Term
-__HERE__
+	${PYTHON3} ${SCRIPT_DIR}/lib/csv-to-xlsx.py \	
```

Comment:
remove spaces after "\" characters (also in following lines)

Reply:
This should be fixed by the patch.